src/libpgmoneta/wf_backup.c

```diff
@@ -47,9 +47,6 @@
 static char* basebackup_name(void);
 static int basebackup_execute(char*, struct art*);
 
-static int send_upload_manifest(SSL* ssl, int socket);
```

Comment:
These are a lot of breaking changes. Make sure that base backup and incr backup for > 17 still works.